python/py_mpi.cpp

```diff
@@ -45,5 +45,6 @@ void setupMpi(py::module& m)
   py::class_<bufr::mpi::Comm>(m, "Comm")
     .def(py::init<const std::string&>())
     .def("name", &bufr::mpi::Comm::name)
-    .def("rank", &bufr::mpi::Comm::rank);
+    .def("rank", &bufr::mpi::Comm::rank)
+    .def("size", &bufr::mpi::Comm::size);
```

Comment:
How was "size" not necessary before?

Reply:
I just never ended up using it, so never noticed it was missing. Should have been there all along..

Reply:
I don't know where it could be done, but some documentations of the difference between "rank" and "size" might be helpful.